src/components/StyledModal.js

```diff
@@ -82,6 +82,13 @@ export const ModalHeader = ({ children, onClose }) => (
   </Header>
 );
 
+ModalHeader.propTypes = {
+  /** handles how the modal is closed */
+  onClose: PropTypes.func.isRequired,
+  /** children */
+  children: PropTypes.func,
```

Comment:
Should `children` be a `node`?

Reply:
Yes, but in the same file has some children properties as "func". I don't know why.

Reply:
Let's correct that then.

Reply:
Done.